### PR TITLE
Feature/current openads log system

### PR DESCRIPTION
**Pull request by alextremp**
# Background

OpenAds and the connectors are using the LogLevel dependency in order to create the Log instance.
In order to reduce dependencies and code duplications to use loglevel in each component, this library will offer the resolved Log, DEBUG property and Proxy for debug automation in containers, so each component can include the utilities and just use it.

# Goal

* Be used in any OpenAds connector, and in OpenAds too.
* Remove all code duplications from current developed components.
* Remove the need of declaring that a Connector implements a Logger interface (connector api)

# Implementation

* Log singleton is accessible by lib clients to log messages manually
* DEBUG property is accessible by lib clients to know when OPENADS will be initialized in DEBUG mode (so the 3rd party libs that can be run in debug mode, p.ex. apntag, should be enabled in debug mode) 
* debugInstance function is accessible by lib clients in order to proxy objects that will debug each access on them.

# Further considerations

After publishing this version, a non-breaking change update should be done in each component.

# Checklist

- [X] The PR relates to *only* one subject with a clear title.
- [X] I have performed a self-review of my own code
- [X] Wrote [good commit messages](http://chris.beams.io/posts/git-commit/)
- [X] My code is readable by someone else, and I commented the hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works

# Memetized Description

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@schibstedspain/openads-logger",
+  "version": "1.0.0",
+  "description": "OpenAds Logger: Log utility for OpenAds and its connectors",
+  "main": "dist/",
+  "scripts": {
+    "cleanDistFolder": "rm -Rf ./dist",
```
Comment (rafamarquezv) [inline package.json:7]: We should use "clean-webpack-plugin" dependency which work under Windows too.

**Review by rafamarquezv — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rafamarquezv) [reply package.json:7]: Ok, it has no webpack, my bad.

**Review by rafamarquezv — APPROVED** [review]

**Review by pablovegau — APPROVED** [review]

**Pull request merged**